run-tests.php

```diff
@@ -305,6 +305,12 @@ function main(): void
         'opcache.jit_hot_func=1',
         'opcache.jit_hot_return=1',
         'opcache.jit_hot_side_exit=1',
+        'opcache.jit_max_root_traces=100000',
```

Comment:
Do we actually need this? Tests are small, it's unlikely we're hitting `jit_max_root_traces`. Did you verify this for any test?

Reply:
I'd like to use the same parameters everywhere, to avoid hitting any limit and not compiling any trace, in case the unit tests were to include some bigger integration tests in the future.

Reply:
This setting may disable some tests in case they set a small ``opcache.jit_buffer_size``.

Reply:
@dstogov Updated the jit_buffer_size to be the same in all tests.